### PR TITLE
Add 1.0.1 and Question FAQ

### PR DESCRIPTION
**Pull request by Cyber-Thibaut**
Added information about the 1.0.1 maj and added the question about the disappearance of the USB menu in the settings from the FAQ

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -344,6 +344,26 @@
         "Atomic": "Bug corrected in the periodic table",
         "Gutter": "Gutter corrected in the python application"
       }
+      "1-0-1": {
+        "XNT" : "XNT Cyclique (Plusieurs appuis sur XNT changent le caractère inséré entre X, N, T et θ)",
```
Comment (Yaya-Cout) [inline src/locales/en.json:348]: Can you translate it ?

File: src/locales/en.json [hunk 2]
```diff
@@ -344,6 +344,26 @@
...
+        "LastApp" : "Shortcut Shift + Ans to go to the last used application",
+        "CorrPix" : "Fixed pixel shift on some icons in some themes",
+        "Dim" : "Dimension in additional results when a unit is used",
+        "PyMaJ" : "MicroPython update from 1.17 to 1.19.1",
+        "PyCrash" : "Fixed crash when opening scripts",
+        "PyItal" : "Italicized text in strings",
+        "AddUlab" : "Added `ulab.utils`.",
+        "PyLightControl" : "Control of brightness via `set_brightness` and `get_brightness`.",
+        "AddStat" : "Geometric mean, Harmonic mean and Mode in the Stats tab",
+        "AddFunctionLiseuse" : "Additional functions in TRU (sums, products, integrals and binomial coefficients)",
```
Comment (Yaya-Cout) [inline src/locales/en.json:361]:
```suggestion
        "AddFunctionLiseuse" : "Additional functions in URT (sums, products, integrals and binomial coefficients)",
```

File: src/locales/en.json [hunk 3]
```diff
@@ -344,6 +344,26 @@
...
+        "AddCaracURT" : "Extra characters in URT (``f` and ``i`)",
+        "CorrBrownColor" : "Brown color correction in URT",
+        "CorrErrorBack" : "Fix syntax error when reverting",
+        "BlockHardware" : "Blocking of hardwares tests in exam mode",
```
Comment (Yaya-Cout) [inline src/locales/en.json:365]: Can you rename to `BlockHardwareTests` ?
```suggestion
        "BlockHardware" : "Blocking of hardwares tests in exam mode"
```

File: src/locales/en.json [hunk 4]
```diff
@@ -344,6 +344,26 @@
...
+        "PyMaJ" : "MicroPython update from 1.17 to 1.19.1",
+        "PyCrash" : "Fixed crash when opening scripts",
+        "PyItal" : "Italicized text in strings",
+        "AddUlab" : "Added `ulab.utils`.",
+        "PyLightControl" : "Control of brightness via `set_brightness` and `get_brightness`.",
+        "AddStat" : "Geometric mean, Harmonic mean and Mode in the Stats tab",
+        "AddFunctionLiseuse" : "Additional functions in TRU (sums, products, integrals and binomial coefficients)",
+        "AddCaracURT" : "Extra characters in URT (``f` and ``i`)",
+        "CorrBrownColor" : "Brown color correction in URT",
+        "CorrErrorBack" : "Fix syntax error when reverting",
```
Comment (Yaya-Cout) [inline src/locales/en.json:364]: Can you explain that it's in the reader app ?

File: src/locales/en.json [hunk 5]
```diff
@@ -344,6 +344,26 @@
...
+        "NewPol" : "Changed font",
+        "TxtItal" : "Italicized text in some places such as function definitions in the functions application and texts and comments in the Python editor",
+        "TxtTool" : "Larger text in toolboxes with automatic scrolling when text is too long",
+        "LastApp" : "Shortcut Shift + Ans to go to the last used application",
+        "CorrPix" : "Fixed pixel shift on some icons in some themes",
+        "Dim" : "Dimension in additional results when a unit is used",
+        "PyMaJ" : "MicroPython update from 1.17 to 1.19.1",
+        "PyCrash" : "Fixed crash when opening scripts",
+        "PyItal" : "Italicized text in strings",
+        "AddUlab" : "Added `ulab.utils`.",
```
Comment (Yaya-Cout) [inline src/locales/en.json:358]:
```suggestion
        "AddUlab" : "Added `ulab.utils`",
```

File: src/locales/en.json [hunk 6]
```diff
@@ -425,6 +445,10 @@
       "stillCouldntFind": {
         "question": "Couldn't find an answer to your question?",
         "answer": "Go to the <a href=\"https://discord.gg/X2TWhh9\">Omega community discord server</a> and ask there!"
+      },
+      "noUSBmenu": {
+        "question": "The USB protection menu does not appear anymore in the calculator settings, is this normal?",
+        "answer": "The USB protection has been integrated into the bootloader and therefore no longer needs a menu.",
```
Comment (Yaya-Cout) [inline src/locales/en.json:451]:
```suggestion
        "answer": "The USB protection has been integrated into the bootloader and therefore no longer needs a menu."
```

File: src/locales/en.json [hunk 7]
```diff
@@ -477,5 +501,4 @@
     "switch-cable-or-computer": "Try changing cable or PC.",
     "using-chromium-snap": "You are using Chromium in Snap.",
     "using-linux": "You are using Linux."
-  }
-}
\ No newline at end of file
+  }
```
Comment (Yaya-Cout) [inline src/locales/en.json:504]: Why did you remove the last bracket ?

File: src/locales/fr.json [hunk 8]
```diff
@@ -333,6 +333,26 @@
...
+                "PyCrash" : "Correction du crash lors de l'ouverture des scripts",
+                "PyItal" : "Texte en italique dans les chaines de caractères",
+                "AddUlab" : "Ajout de `ulab.utils`",
+                "PyLightControl" : "Contôle de la lumisosité via `set_brightness` et `get_brightness`",
+                "AddStat" : "Moyenne géométrique, Moyenne harmonique et Mode dans l'onglet Stats",
+                "AddFunctionLiseuse" : "Fonctions en plus en URT (Sommes, Produits, Intégrales et coefficients binomiaux)",
+                "AddCaracURT" : "Caractères en plus en URT (`\f` et `\i`)",
+                "CorrBrownColor" : "Correction de la couleur marron en URT",
+                "CorrErrorBack" : "Correction de l'erreur de syntaxe lors du retour en arrière",
+                "BlockHardware" : "Blocage des hardwares tests en mode examen",
```
Comment (Yaya-Cout) [inline src/locales/fr.json:354]: Can you rename it too ?
```suggestion
                "BlockHardware" : "Blocage des hardwares tests en mode examen"
```

**Review by Yaya-Cout — CHANGES_REQUESTED** [review]
Can you explain in which app are the changes ? It is not very clear.

File: src/locales/en.json [hunk 9]
```diff
@@ -344,6 +344,26 @@
         "Atomic": "Bug corrected in the periodic table",
         "Gutter": "Gutter corrected in the python application"
       }
+      "1-0-1": {
+        "XNT" : "XNT Cyclic (Multiple presses of XNT change the character inserted between X, N, T and θ)",
+        "NewPol" : "Changed font",
+        "TxtItal" : "Italicized text in some places such as function definitions in the functions application and texts and comments in the Python editor",
+        "TxtTool" : "Larger text in toolboxes with automatic scrolling when text is too long",
```
Comment (Yaya-Cout) [inline src/locales/en.json:351]: Can you rename it to `ToolboxFontSize` ?
```suggestion
        "TxtTool" : "Larger text in toolboxes with automatic lateral scrolling when text is too long",
```

File: src/locales/en.json [hunk 10]
```diff
@@ -344,6 +344,26 @@
...
+        "LastApp" : "Shortcut Shift + Ans to go to the last used application",
+        "CorrPix" : "Fixed pixel shift on some icons in some themes",
```
Comment (Yaya-Cout) [inline src/locales/en.json:353]: Can you rename it to `FixShiftedIcon` or something similar ?

File: src/locales/en.json [hunk 11]
```diff
@@ -344,6 +344,26 @@
         "Atomic": "Bug corrected in the periodic table",
         "Gutter": "Gutter corrected in the python application"
       }
```
Comment (Yaya-Cout) [inline src/locales/en.json:346]:
```suggestion
      },
```

File: src/locales/en.json [hunk 12]
```diff
@@ -344,6 +344,26 @@
...
+      "1-0-1": {
+        "XNT" : "XNT Cyclic (Multiple presses of XNT change the character inserted between X, N, T and θ)",
+        "NewPol" : "Changed font",
```
Comment (Yaya-Cout) [inline src/locales/en.json:349]: Can you rename to `NewFont` and edit the text to `New font` (I prefer this form, but it's just me)
```suggestion
        "NewPol" : "New font",
```

File: src/locales/en.json [hunk 13]
```diff
@@ -344,6 +344,26 @@
...
+        "TxtItal" : "Italicized text in some places such as function definitions in the functions application and texts and comments in the Python editor",
+        "TxtTool" : "Larger text in toolboxes with automatic scrolling when text is too long",
+        "LastApp" : "Shortcut Shift + Ans to go to the last used application",
+        "CorrPix" : "Fixed pixel shift on some icons in some themes",
+        "Dim" : "Dimension in additional results when a unit is used",
+        "PyMaJ" : "MicroPython update from 1.17 to 1.19.1",
```
Comment (Yaya-Cout) [inline src/locales/en.json:355]: Can you rename it to `MicroPythonUpgrade` ?

File: src/locales/en.json [hunk 14]
```diff
@@ -344,6 +344,26 @@
...
+        "PyCrash" : "Fixed crash when opening scripts",
+        "PyItal" : "Italicized text in strings",
+        "AddUlab" : "Added `ulab.utils`",
+        "PyLightControl" : "Control of brightness via `set_brightness` and `get_brightness`.",
```
Comment (Yaya-Cout) [inline src/locales/en.json:359]: Can you rename it to `PythonBrightnessControl` ?

File: src/locales/en.json [hunk 15]
```diff
@@ -344,6 +344,26 @@
...
+        "AddStat" : "Geometric mean, Harmonic mean and Mode in the Stats tab",
+        "AddFunctionLiseuse" : "Additional functions in URT (sums, products, integrals and binomial coefficients)",
```
Comment (Yaya-Cout) [inline src/locales/en.json:361]: Can you rename it to `AddUrtFunctions` or something similar ?

File: src/locales/en.json [hunk 16]
```diff
@@ -344,6 +344,26 @@
...
+        "AddCaracURT" : "In the reader application, add extra characters in the URT (``f` and ``i`)",
```
Comment (Yaya-Cout) [inline src/locales/en.json:362]: Can you rename it to `AddCharactersUrt`

File: src/locales/en.json [hunk 17]
```diff
@@ -344,6 +344,26 @@
...
+        "CorrBrownColor" : "Brown color correction in URT for the reader application",
```
Comment (Yaya-Cout) [inline src/locales/en.json:363]: Can you rename it to `FixBrownColorUrt` ?

File: src/locales/en.json [hunk 18]
```diff
@@ -344,6 +344,26 @@
...
+        "CorrErrorBack" : "Fixed a syntax error when reverting in the reader application",
```
Comment (Yaya-Cout) [inline src/locales/en.json:364]: Can you rename it to `FixReaderErrorBack` ?

File: src/locales/en.json [hunk 19]
```diff
@@ -344,6 +344,26 @@
...
+        "BlockHardware" : "Blocking of hardwares tests in exam mode"
```
Comment (Yaya-Cout) [inline src/locales/en.json:365]: Can you rename it to `BlockHardwareTests` ?

File: src/locales/en.json [hunk 20]
```diff
@@ -478,4 +502,3 @@
     "using-chromium-snap": "You are using Chromium in Snap.",
     "using-linux": "You are using Linux."
   }
```
Comment (Yaya-Cout) [inline src/locales/en.json:504]: Why did you remove the last bracket ?

File: src/locales/en.json [hunk 21]
```diff
@@ -344,6 +344,26 @@
...
+        "XNT" : "XNT Cyclic (Multiple presses of XNT change the character inserted between X, N, T and θ)",
+        "NewPol" : "Changed font",
+        "TxtItal" : "Italicized text in some places such as function definitions in the functions application and texts and comments in the Python editor",
+        "TxtTool" : "Larger text in toolboxes with automatic scrolling when text is too long",
+        "LastApp" : "Shortcut Shift + Ans to go to the last used application",
+        "CorrPix" : "Fixed pixel shift on some icons in some themes",
+        "Dim" : "Dimension in additional results when a unit is used",
+        "PyMaJ" : "MicroPython update from 1.17 to 1.19.1",
+        "PyCrash" : "Fixed crash when opening scripts",
+        "PyItal" : "Italicized text in strings",
```
Comment (Yaya-Cout) [inline src/locales/en.json:357]: Can you explain that it's in the Python editor ?

**Review by Yaya-Cout — CHANGES_REQUESTED** [review]
Can you also apply changes to the French translation ?

File: src/locales/fr.json [hunk 22]
```diff
@@ -333,6 +333,26 @@
                 "Atomic": "Bug corrigé dans le tableau périodique",
                 "Gutter": "Gouttière corrigée dans l'application python"
             }
```
Comment (Yaya-Cout) [inline src/locales/fr.json:335]:
```suggestion
            },
```

**Comment by Cyber-Thibaut** [comment]
changes applied in all affected files (en/fr and releases.vue)

File: src/locales/fr.json [hunk 23]
```diff
@@ -414,6 +434,10 @@
             "uninstallExternal": {
                 "answer": "Accédez à la <a href=\"https://upsilonnumworks.github.io/Upsilon-External\">page des applications externes</a> en ne sélectionnant rien, ensuite, cliquez sur installer. Cela effacera toutes les applications externes ainsi que l'arrière-plan.",
                 "question": "Comment désinstaller des applications externes ?"
+            },
+            "noUSBmenu": {
+                "answer": "La protection USB a été intégrée au bootloader et n'as donc plus besoin de menu.",
```
Comment (Yaya-Cout) [inline src/locales/fr.json:439]:
```suggestion
                "answer": "La protection USB a été intégrée au bootloader et n'a donc plus besoin de menu.",
```

File: src/locales/en.json (hunk 10 above)
Comment (Yaya-Cout) [reply src/locales/en.json:353]: Why did you mark it as resolved ?

File: src/locales/en.json [hunk 24]
```diff
@@ -343,7 +343,27 @@
         "RPN": "Fix bugs in RPN",
         "Atomic": "Bug corrected in the periodic table",
         "Gutter": "Gutter corrected in the python application"
-      }
+      },
+      "1-0-1": {
+        "XNT" : "XNT Cyclic (Multiple presses of XNT change the character inserted between X, N, T and θ)",
```
Comment (Yaya-Cout) [inline src/locales/en.json:348]:
```suggestion
        "CyclicXNT" : "XNT Cyclic (Multiple presses of XNT change the character inserted between X, N, T and θ)",
```

File: src/locales/en.json [hunk 25]
```diff
@@ -343,7 +343,27 @@
...
+        "NewFont" : "New font",
+        "TxtItal" : "Italicized text in some places such as function definitions in the functions application and texts and comments in the Python editor",
```
Comment (Yaya-Cout) [inline src/locales/en.json:350]:
```suggestion
        "AddItalize" : "Italicized text in some places such as function definitions in the functions application and texts and comments in the Python editor",
```

File: src/locales/en.json [hunk 26]
```diff
@@ -343,7 +343,27 @@
...
+        "ToolboxFontSize" : "Larger text in toolboxes with automatic lateral scrolling when text is too long",
+        "FixShiftedIcon" : "Shortcut Shift + Ans to go to the last used application",
```
Comment (Yaya-Cout) [inline src/locales/en.json:352]:
```suggestion
        "ShiftAns" : "Shortcut Shift + Ans to go to the last used application",
```

File: src/locales/en.json [hunk 27]
```diff
@@ -343,7 +343,27 @@
...
+        "CorrPix" : "Fixed pixel shift on some icons in some themes",
```
Comment (Yaya-Cout) [inline src/locales/en.json:353]:
```suggestion
        "FixShiftedIcon" : "Fixed pixel shift on some icons in some themes",
```

File: src/locales/en.json [hunk 28]
```diff
@@ -343,7 +343,27 @@
...
+        "Dim" : "Dimension in additional results when a unit is used",
```
Comment (Yaya-Cout) [inline src/locales/en.json:354]:
```suggestion
        "DimInAdditionalsResults" : "Dimension in additional results when a unit is used",
```

File: src/locales/en.json [hunk 29]
```diff
@@ -343,7 +343,27 @@
...
+        "MicroPythonUpgrade" : "MicroPython update from 1.17 to 1.19.1",
+        "PyCrash" : "Fixed a crash when opening scripts in the Python editor",
```
Comment (Yaya-Cout) [inline src/locales/en.json:356]:
```suggestion
        "PythonCrash" : "Fixed a crash when opening scripts in the Python editor",
```

File: src/locales/en.json (hunk 18 above)
Comment (Yaya-Cout) [reply src/locales/en.json:364]:
```suggestion
        "FixReaderErrorBack" : "Fixed a syntax error when reverting in the reader application",
```

**Review by Yaya-Cout — CHANGES_REQUESTED** [review]
Can you apply these changes and reflect the on others files ?

**Pull request merged**